geode-junit/src/main/java/org/apache/geode/codeAnalysis/AnalyzeDataSerializablesTestBase.java

```diff
@@ -230,10 +222,18 @@ private void serializeAndDeserializeObject(Object object) throws Exception {
     BufferDataOutputStream outputStream = new BufferDataOutputStream(KnownVersion.CURRENT);
     try {
       serializeObject(object, outputStream);
-    } catch (IOException e) {
+    } catch (IOException | NullPointerException e) {
       // some classes, such as BackupLock, are Serializable because the extend something
       // like ReentrantLock but we never serialize them & it doesn't work to try to do so
       System.out.println("Not Serializable: " + object.getClass().getName());
+      return;
+    } catch (Exception e) {
```

Comment:
Why not catch `CacheClosedException` here, instead of `Exception`?

Reply:
AnalyzeMembershipSerializablesIntegrationTest extends AnalyzeDataSerializablesTestBase and lives in geode-membership which does not have a dependency on geode-core. So when that test runs, it was throwing ClassNotFoundException for CacheClosedException. Then I removed ClassNotFoundException and changed it to Exception with the dreaded type check.